signaling-server/setup.py.in

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup
+
+
+def read_file(name):
+    with open(name, "r", encoding="utf-8") as fh:
+        return fh.read()
+
+
+def read_file_lines(name):
+    with open(name, "r", encoding="utf-8") as fh:
+        return fh.readlines()
+
+
+long_description = read_file("README.md")
+requirements = read_file_lines("requirements.txt")
+
+setup(
+    name="opentera_webrtc_signaling_server",
+    version="${OPENTERA_WEBRTC_NATIVE_CLIENT_VERSION}",
+    author="Marc-Antoine Maheux, Cedric Godin, Dominic Letourneau",
+    author_email="marc-antoine.maheux@usherbrooke.ca, cedric.godin@usherbrooke.ca, dominic.letourneau@usherbrooke.ca",
+    description="OpenTera WebRTC Signaling Server",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/introlab/opentera-webrtc",
+    packages=['opentera.webrtc.signaling_server'],
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "Development Status :: 4 - Beta",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: POSIX",
+    ],
+    license="Apache Software License",
+    platforms=['linux_x86_64'],
+    python_requires='>=3.8',
+    zip_safe=False,
+    install_requires=requirements,
+    scripts=['opentera/webrtc/signaling_server/signaling_server.py'],
+)
```

Comment:
Add installation to signaling server.
Currently uses the same version as `native_client`. Should they have unrelated versioning?

Reply:
I think it's a good idea to have matching version!